### PR TITLE
add methods to SpanId and TraceId to make id conversion simpler.

### PR DESCRIPTION
**Pull request by jkwatson**

### CODE REVIEW

File: api/src/main/java/io/opentelemetry/trace/SpanId.java [hunk 1]
```diff
@@ -107,6 +107,21 @@ public static SpanId fromBytes(byte[] src, int srcOffset) {
...
 
+  /**
+   * Transforms a byte[] representation of a span id (eg. from a protobuf) and returns a base 16
+   * representation of the same.
+   *
+   * @param bytes The byte array containing the span id.
+   * @return A base 16 representation of the span id.
+   * @since 0.1.0
+   */
+  public static String asBase16(byte[] bytes) {
```
Comment (breedx-nr) [inline api/src/main/java/io/opentelemetry/trace/SpanId.java]: What can I do to talk you into a nice `toHexString()` today?

Comment (jkwatson) [reply api/src/main/java/io/opentelemetry/trace/SpanId.java]: I'm going to make this match the rest of the methods in the class more closely, and rename it to `asLowerBase16`.

**Comment by jkwatson** [comment]
ok, ready for a re-review

**Review by breedx-nr — APPROVED** [review]
Oh sure, I suppose naming consistency is good.

**Pull request merged**